components/ui/FormattedDate.tsx

```diff
@@ -25,16 +25,51 @@ export function FormattedDate({
 }) {
   const oneWeekMs = 1000 * 60 * 60 * 24 * 7
 
+  const wrapFormattedDate = useCallback(
+    (formattedDate: string) => {
+      return (
+        <InfoButton
+          tooltip={`${getDateYYYYMMDD(date!)}${
+            includeTime ? ` at ${date!.toLocaleTimeString()}` : ""
+          }`}
+          showInfoButton={false}
+        >
+          <span className={clsx(className)} suppressHydrationWarning={true}>
+            {prefix}
+            {capitalise
+              ? formattedDate.charAt(0).toUpperCase() + formattedDate.slice(1)
+              : formattedDate}
+            {postfix}
+          </span>
+        </InfoButton>
+      )
+    },
+    [capitalise, className, date, includeTime, postfix, prefix],
+  )
+
   if (!date || typeof date.getTime !== "function" || isNaN(date.getTime())) {
     return <></>
   }
 
-  const showDistance =
-    alwaysUseDistance ||
+  const isRecent =
```

Comment:
`isRecent` is a bit misleading, it's either: 
	1.	In the past but within the last 8 weeks, or
	2.	In the future but within one week from now